src/pykeen/datasets/base.py

```diff
@@ -944,3 +945,105 @@ def _get_df(self) -> pd.DataFrame:
             df = df[usecols]
 
         return df
+
+
+class HyperRelationalUnpackedRemoteDataset(PathDataset):
```

Comment:
I think we need a different base class mixin for this kind of dataset to note that the triples factories have to be SatementFactories